daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/EvaluationOrderTest.scala

```diff
@@ -201,7 +201,7 @@ class EvaluationOrderTest extends AnyFreeSpec with Matchers with Inside {
 
       val exercise_interface: Party -> ContractId M:Person -> Update Unit =
         \(exercisingParty: Party) (cId: ContractId M:Person) ->
-          Test:run @Int64 (exercise_by_interface @M:Person Nap cId 42 (\(x: M:Person) -> TRACE @Bool "interface guard" True));
+          Test:run @Int64 (exercise_interface_with_guard @M:Person Nap cId 42 (\(x: M:Person) -> TRACE @Bool "interface guard" True));
```

Comment:
should we extend that test with something without a guard or do you rely on that being equivalent to a guard that is `const True`?